playground/vite.config.ts

```diff
@@ -5,7 +5,16 @@ import tailwindcss from 'tailwindcss';
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [vue()],
+  plugins: [
+    vue({
+      template: {
+        compilerOptions: {
+          isCustomElement: (tag) => tag.includes('-ce')
+        }
+      },
+      customElement: /^(?!.*App\.vue).*$/
+    })
+  ],
```

Comment:
Don't forget to remove this config in the playground it will cause issues when using vue components instead of web components